src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -129,14 +139,17 @@ public JedisSentinelPool(String masterName, Set<String> sentinels,
       final int sentinelConnectionTimeout, final int sentinelSoTimeout, final String sentinelPassword,
       final String sentinelClientName) {
     this(masterName, sentinels, poolConfig, connectionTimeout, soTimeout, null, password, database, clientName,
-        sentinelConnectionTimeout, sentinelSoTimeout, null, sentinelPassword, sentinelClientName);
+        sentinelConnectionTimeout, sentinelSoTimeout, null, sentinelPassword, sentinelClientName, false, false,
+        null, null, null);
```

Comment:
After keeping previously existing constructor, this change won't be required.

Reply:
reverted the change